src/test/rustdoc/nested-modules.rs

```diff
@@ -0,0 +1,17 @@
+#![crate_name = "foo"]
+
+mod a_module {
+  fn private_function() {
+      println!("called `a_module::private_function()`");
+  }
+
```

Comment:
(You may have to adjust the test until it passes.)

```suggestion

// @has 'foo/a_module/index.html' '//a[@href="fn.private_function.html"]' 'other_private_function'
use private_function as other_private_function;

```